login-service/static/login-debug/index.html

```diff
@@ -22,7 +22,7 @@
             white-space: nowrap;
         }
     </style>
-    <script src="/vue/dist/vue.global.prod.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vue@3.4.31/dist/vue.global.prod.js"></script>
```

Comment:
possible to specify only part of the version to allow future vue patches in case of security vulnerability?

Reply:
changed with 26367fd4abd44b5313889594c26ba3aa91866aa4